ui/i18n/src/main/res/values-ar/strings.xml

```diff
@@ -86,6 +86,9 @@
   <string name="feed_auto_download_global">اعدادات افتراضية شاملة</string>
   <string name="feed_auto_download_always">دائما</string>
   <string name="feed_auto_download_never">ابدا</string>
+  <string name="feed_skip_inbox_global">اعدادات افتراضية شاملة</string>
```

Comment:
This must have been a lot of work. Unfortunately, we cannot merge translation changes, as they might conflict with changes that are done on Transifex. Please only change the English strings file.

(we already mention this [here](https://github.com/AntennaPod/AntennaPod/blob/develop/CONTRIBUTING.md#submit-a-pull-request) and the URL is displayed in the default PR description. If you have an idea on how to make this more clear, feel free to do a PR)

Reply:
OK. I only made changes using known values from elsewhere in the file, and then in the languages which I am (at least moderately) fluent in. I purposely made all the other changes in a separate commit so it can be reverted easily.

Reply:
The translation file changes are reverted. Sorry for the oversight.